### PR TITLE
Add default issue template

### PR DESCRIPTION
**Pull request by dnsmichi**
Partially touches https://github.com/XollabOS/talktometechnology/issues/6 - but a simple version with Markdown.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/default.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+## Proposal
+
+<!-- Share the proposed changes, or go ahead with creating a new merge request and linking it here. -->
+
+## Resources
+
+## Tasks
+
+- [ ] ...
```
Comment (emmalearnscode) [inline .github/ISSUE_TEMPLATE/default.md:13]:
```suggestion
- [ ] task 1
- [ ] task 2
```

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by emmalearnscode — COMMENTED** [review]
I would suggest adding a section for screenshots too.

File: .github/ISSUE_TEMPLATE/default.md (hunk 1 above)
Comment (dnsmichi) [reply .github/ISSUE_TEMPLATE/default.md:13]: Thanks, great suggestion :)

File: .github/ISSUE_TEMPLATE/default.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+## Problem to solve
+
+<!-- Explain the problem in your own words. -->
+
+## Proposal
+
+<!-- Share the proposed changes, or go ahead with creating a new merge request and linking it here. -->
+
+## Resources
+
```
Comment (dnsmichi) [inline .github/ISSUE_TEMPLATE/default.md:10]: @emmalearnscode I'd add it into Resources :) 

```suggestion
### Screenshots

```

**Review by emmalearnscode — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
LGTM, In the future we can change from Markdown to YAML ([`issue forms`](https://github.blog/changelog/2021-06-23-issues-forms-beta-for-public-repositories/)) config

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Efficient, Maintainable

**Review by vikasganiga05 — APPROVED** [review]
LGTM :+1:

**Comment by dnsmichi** [comment]
@Pradumnasaraf Thanks, good idea. I thought of that but never have done the YAML version before.

**Comment by Pradumnasaraf** [comment]
> @Pradumnasaraf Thanks, good idea. I thought of that but never have done the YAML version before.

Markdown is good for the start, in future someone will update that  : )

**Pull request merged**